README.md

```diff
@@ -65,3 +65,36 @@ available charm run:
 If you aim to use Katib within an existing Kubeflow deployment in order to use it within the Kubeflow dashboard, you will have to integrate `katib-ui` to `istio-pilot` with the following command:
 
     juju relate istio-pilot katib-ui
+
+## Setting Custom Images for Katib Controller
+
+Katib controller comes with a set of preconfigured images that are used in Katib workloads. These images are part of the charm configuration under `images` in the `charms/katib-controller/config.yaml` file. The following is a list of default images used:
+
+```json
```

Comment:
Can we instead work with YAML files? This is closer to both what K8s is using and what people are accustomed to more

Reply:
It works also with yaml and also with json the config value description is saying that